alf/algorithms/algorithm.py

```diff
@@ -470,46 +471,53 @@ def _setup_optimizers_(self):
             tuple:
 
             - list of parameters not handled by any optimizers under this algorithm
```

Comment:
This sentence might be ambiguous. It should be "list of parameters not handled by optimizers of child modules but handled by this algorithm itself"

Reply:
they are not handled by this algorithm either.